### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by danifromecuador**
Last part of the Portolio's mobile version using a form tag and flexbox

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -223,6 +224,21 @@ <h4>LET'S CONNECT</h4>
     </ul>
 
   </section>
+  <section class="contact">
+    <header>Contact me</header>
+    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it </p>
+    <form action="https://formspree.io/f/xnqrekyo" method="POST">
+      <label for="text"></label>
+      <input type="text" id="text" name="name" placeholder="name:" required maxlength="30">
```
Comment (huseyinbiyik) [inline index.html:232]: Good job @danifromecuador. But kindly stick to the design and fix the padding according to the Figma template.
Right now your design has 12px padding.
Screenshot 2022-12-15 at 1 36 54 AM

Figma paddings are slightly different.
Screenshot 2022-12-15 at 1 30 54 AM

File: index.html [hunk 2]
```diff
@@ -223,6 +224,21 @@ <h4>LET'S CONNECT</h4>
...
+      <label for="email"></label>
+      <input type="email" id="email" name="email" placeholder="email:" required>
```
Comment (huseyinbiyik) [inline index.html:234]: Again, kindly follow the Figma template for all padding and margins and all sizes.
Your code:
Screenshot 2022-12-15 at 1 40 38 AM


Figma template:
Screenshot 2022-12-15 at 1 31 08 AM

File: index.html [hunk 3]
```diff
@@ -223,6 +224,21 @@ <h4>LET'S CONNECT</h4>
...
+      <label for="textarea"></label>
+      <textarea name="textarea" id="textarea" placeholder="your message here:" required maxlength="500"></textarea>
```
Comment (huseyinbiyik) [inline index.html:236]: Kindly follow the design for the paddings and font size. Your font size is 17px and Figma template is 15px.

Your code has slightly different padding
Screenshot 2022-12-15 at 1 42 44 AM



than the Figma template.
Screenshot 2022-12-15 at 1 28 18 AM

File: index.html [hunk 4]
```diff
@@ -223,6 +224,21 @@ <h4>LET'S CONNECT</h4>
...
+      <label for="button"></label>
+      <button  type="submit" id="button">Get In Touch</button>
```
Comment (huseyinbiyik) [inline index.html:238]: Kindly add the hover color following the color defined in Figma template.
Figma template:
Screenshot 2022-12-15 at 1 23 29 AM

Your design:
Screenshot 2022-12-15 at 1 22 56 AM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @danifromecuador,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Commit messages are good 👌
- The design is implemented very well. 👨‍🎨

## Required Changes ♻️
- Your job is fantastic. But we should always stick to the design provided. Check the inline comments below.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-[OPTIONAL] To delete the part below on your Readme. That message is a note for the student.
Screenshot 2022-12-15 at 1 51 47 AM
-[OPTIONAL] Kindly move your images onto a folder. That will help your repo to look more professional and keep it more organized. 📂


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @danifromecuador  , I'm @MaganaAsiati , your reviewer for this PR  :wave: 
Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 





## Highlights
- Descriptive pull request :heavy_check_mark:👨🏻‍💻
- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :+1: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
Cheers and Happy coding! 🤓 👏👏👏

**Pull request merged**